recipes/maicos/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "maicos" %}
+{% set version = "0.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 89fa29effb1e384e1faf91a8fabb252d5779206ea3553578aa2cc2e2d4cb6279
+
+build:
+  number: 0
+  skip: true  # [python_impl == 'pypy']
+  skip: true  # [py>310]
+  skip: true  # [py<39]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - cython                                 # [build_platform != target_platform]
+    - numpy                                  # [build_platform != target_platform]
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - libgomp  # [linux]
+  host:
+    - python
+    - pip
+    - numpy {{ numpy }}
+    - cython
+    - packaging
+
+  run:
+    - python
+    - scipy >=1.0.0
+    - {{ pin_compatible('numpy', lower_bound='1.16') }}
+    - mdanalysis >=2.2.0
+    - mdacli >=0.1.12
+
+
+test:
+  imports:
+    - MDAnalysis
+    - MDAnalysis.analysis
+
+about:
+  home: https://www.maicos-analysis.org
+  license: GPL-3.0-or-later
+  license_family: GPL
+  license_file: LICENSE
+  summary: A Python package for Molecular Analysis for Interfacial and Confined Systems.
+  description: MAICoS is an object-oriented python toolkit for analysing the structure and 
+    dynamics of interfacial and confined fluids from molecular simulations. Combined with 
+    MDAnalysis, MAICoS can be used to extract density profiles, dielectric constants, 
+    structure factors, or transport properties from trajectories files, including 
+    LAMMPS, GROMACS, CHARMM or NAMD data. MAICoS is open source and is released under the 
+    GNU general public license v3.0.
+  doc_url: https://www.maicos-analysis.org
+  dev_url: https://gitlab.com/maicos-devel/maicos
+
+extra:
+  recipe-maintainers:
+    - srihas29
+    - PicoCentauri
+    - hejamu
```

Comment:
@PicoCentauri and @hejamu we need you to comment that you are OK to maintain this recipe.

Reply:
I am also okay with maintaining!